CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Yii HTTP Application Change Log
 
-## 1.0.1 under development
+## 1.1.0 under development
```

Comment:
I think it is still a patch, not a minor. Nothing's broken in the end user API.

Reply:
OK.